ecl2df/wellcompletiondata.py

```diff
@@ -228,7 +228,7 @@ def _df2pyarrow(dframe: pd.DataFrame) -> pyarrow.Table:
         else:
             field_metadata = {}
         if colname == "DATE":
-            dtype = pyarrow.timestamp("ms")
+            dtype = pyarrow.timestamp("ns")
```

Comment:
Careful with this, with nanoseconds you will hit a limit in year 2262, and summary files going past 2262 is something we need. Why is this needed for pandas2?

Reply:
Good point. 👍 

From what I can see, pyarrow 12.0.0 `to_pandas()` method will convert `DATE` column to `datetime64[ns]` despite what we set on the line that you point above. The new pyarrow 14.0.0 does a proper conversion and causing a `dtype` issue in the test below since pandas by default will set dtype to `datetime64[ns]`. I think we can just get away with it by specifying `check_dtype=False` 

https://github.com/equinor/ecl2df/blob/798ee6225697919cad4876d2390e3aa209ba48e0/tests/test_wellcompletiondata.py#L89

Should we ensure that all date column from ecl2df will be using `datetime64[ms]` ?

Reply:
There are tests regarding year 2262 support, check that these cover this case (export to arrow format)